amxmodx/endianness.cpp

```diff
@@ -0,0 +1,18 @@
+#include <endianness.h>
+
+/**
+* Retrieves system endianness.
+*/
+sysEndianness getSysEndianness(void)
```

Comment:
Better to get the system endianness (byte order) while processing data in this case,